pyccl/cosmology.py

```diff
@@ -471,6 +481,11 @@ def _build_parameters(
         c2_mg = self.mg_parametrization.c2_mg
         lambda_mg = self.mg_parametrization.lambda_mg
 
+        if(c1_mg != 0. or c2_mg !=0. or lambda_mg != 0.):
```

Comment:
The default values of c1_mg and c2_mg are actually 1. GR is recovered when c1_mg = c2_mg=1 (regardless of lambda value) or when c1_mg=c2_mg=lambda_mg=0. So, need a slightly more complicated check here to cover both cases. This is why this branch is currently failing a bunch of unit tests I think.

Reply:
@c-d-leonard I've modified the code to only allow c1_mg = c2_mg = 1 or c1_mg = c2_mg = lambda_mg = 0., but complain otherwise with a better error message as you suggested. To be honest, I still had a few unit test problems, but from what I could see, they were not related to this (I think). Let me know if this is fine